internal/ci/github/trybot.cue

```diff
@@ -31,19 +31,22 @@ trybot: _base.#bashWorkflow & {
 
 	on: {
 		push: {
-			branches: [_#defaultBranch] // do not run PR branches
-		}
-		repository_dispatch: {}
-		pull_request: {
-			branches: ["**"]
+			branches: ["trybot/*/*", _#defaultBranch, _base.#testDefaultBranch] // do not run PR branches
 		}
+		pull_request: {}
```

Comment:
I think we can omit this?

Reply:
Why? We need and want the trybot workflow to run for pull requests.

Reply:
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword.

i.e. we only need to set this object, with some of its fields, if we want something other than the default behavior.

Reply:
Resolving per offline conversation.